src/QsCompiler/Transformations/IntrinsicResolutionTransformation.cs

```diff
@@ -9,9 +9,9 @@
 using Microsoft.Quantum.QsCompiler.SyntaxTree;
 
 
-namespace Microsoft.Quantum.QsCompiler.Transformations.IntrinsicResolutionTransformation
+namespace Microsoft.Quantum.QsCompiler.Transformations.IntrinsicResolution
```

Comment:
Should the file be renamed to IntrinsicResolution.cs as well? The naming convention in this folder seems split between files that end in Transformation and those that don't. Since the namespaces are being shortened it could make sense to shorten the filenames too.

Reply:
My preference would be to rename the files, too, dropping the *Transformation ending. It seems silly to have all files within a folder named Transformations ending in Transformations as well. The reason they were named this way I believe is that there is a RewriteStep folder in the Compiler project that contains files with the same name which implement the IRewriteStep interface invoking these transformations. Personally, I think having both files have the same name is fine.

Reply:
Oh, I see. Yes, I think it's fine to have the same filenames since it should be clear from context which one is the transformation. If we don't do that then for consistency I'd argue that Conjugations.cs, FunctorGeneration.cs, etc. should be renamed to ConjugationsTransformation.cs, FunctorGenerationTransformation.cs, etc. I prefer the shorter names. :)

Then we also have files like BasicTransformation.cs and CodeTransformations.cs, but having files named Basic.cs and Code.cs would probably be confusing.